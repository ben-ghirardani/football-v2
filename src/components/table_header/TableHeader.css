@media only screen and (max-width: 320px) {
  th {
    font-size: 11px;
  }
}

@media only screen and (max-width: 400px) {
  th {
    font-size: 12px;
  }
}

.pos {
  text-align: center;
}