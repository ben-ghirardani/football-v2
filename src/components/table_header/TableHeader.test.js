import React from 'react';
import TableHeader from './TableHeader.js';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('TableHeader', ()=> {

  let wrapper;

  beforeEach( ()=>{
    wrapper = shallow(<TableHeader/>);
  });

  it('TableHeader renders as expected', ()=> {
    expect(wrapper).toMatchSnapshot();
  })

  it('renders the correct number and type of HTML elements', ()=> {
    expect(wrapper.find('tr')).toHaveLength(1);
    expect(wrapper.find('th')).toHaveLength(10);
  });

});