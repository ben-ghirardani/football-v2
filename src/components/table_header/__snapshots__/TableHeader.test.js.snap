// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader TableHeader renders as expected 1`] = `ShallowWrapper {}`;
